prisma-local/schema.prisma

```diff
@@ -277,7 +277,6 @@ enum ClientPaymentStatus {
 }
 
 model ClientPayment {
-  id                   String                 @id @default(dbgenerated("(uuid())"))
   paymentId            String                 @unique
```

Comment:
This does not work. You have to specify one @id, so you can change this @unique here in paymentId to @id.

Then don't edit the migration manually, delete it and run prisma migrate dev to generate it. Right now this scheme is out of sync with the migration file so it does not build.